src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubConfiguration.cs

```diff
@@ -43,27 +43,18 @@ public class EventHubConfiguration : IExtensionConfigProvider
         /// </summary>
         public const string LeaseContainerName = "azure-webjobs-eventhub";
 
-        /// <summary>
-        /// default constructor. Callers can reference this without having any assembly references to service bus assemblies. 
-        /// </summary>
-        public EventHubConfiguration()
-            : this(null)
-        {
-        }
-
         /// <summary>
         /// Constructs a new instance.
         /// </summary>
-        /// <param name="options">The optional <see cref="EventProcessorOptions"/> to use when receiving events.</param>
-        public EventHubConfiguration(EventProcessorOptions options)
```

Comment:
Making the options a get/set property, symmetric with ServiceBusConfiguration.MessageOptions. All our other config classes have default constructors with settable properties.